### DISCUSSION TITLE
Helper Scripts In Debian 12?

### DISCUSSION THREAD

**Discussion by Florida-Duke** [tteck/Proxmox#2199] [Help Needed] [answered]
Hello,
First let me say what a great help your Helper Scripts are.

I had used several of the when I was running Proxmox on Bare-Metal (as they say).
Now I have had to install Proxmox ontop of Debian 12 do to computer and driver issues.
Can I still use your scripts in Proxmox even with it running ontop of Debian 12?

Thanks,

**Comment by tteck** [comment]
I would think so, test them, and report back.

**Reply by Florida-Duke** [reply, marked as answer]
Well I'm happy to say that I used your postinstall helper and the chaos helper and both seem to work fine. 
I just have to figure out why following the install guide (not from you) to install Proxmox on top of Debian 12 does create a 'local.lve' storage. Again your scripts seem to work fine.
